x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/new_index/constants.ts

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+// This is indented the way it is to work with ApiCodeExample.
+// Use dedent() when calling this alone
+export const DOCUMENTS_API_JSON_EXAMPLE = `[
```

Comment:
What if you did something like `JSON.stringify(theExampleButLiterallyJSON, null, 2)`